printer_zpl2/views/printing_label_zpl2.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright 2016 SYLEAM
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <menuitem id="base_report_to_printer.printing_menu_label" parent="base_report_to_printer.printing_menu" name="Labels" sequence="50"/>
```

Comment:
The goal was to "provide" a unique menu for labels.
If another module wants to provide labels, it will create a new menu (in this case, this "Labels" menu is useless), or need to depend on printer_zpl2...

How could we achieve this ? Add the "Labels" menu in the base_report_to_printer module instead ?